src/test-utils/channel-fakes/override-messaging-from-main-to-window.ts

```diff
@@ -48,9 +47,7 @@ export const overrideMessagingFromMainToWindow = (mainDi: DiContainer) => {
           );
         }
 
-        const message = tentativeParseJson(data[0]);
```

Comment:
Hmm. I wonder why `tentativeParseJson` is not required anymore. It would be important to still have serialisation and parsing of data over IPC as part of unit testing.